api/groups/urlParams.go

```diff
@@ -6,12 +6,10 @@ import (
 
 	"github.com/gin-gonic/gin"
 	"github.com/multiversx/mx-chain-core-go/core"
+
 	"github.com/multiversx/mx-chain-proxy-go/common"
 )
 
-// SystemAccountAddressBech is the const for the system account address
-const SystemAccountAddressBech = "erd1lllllllllllllllllllllllllllllllllllllllllllllllllllsckry7t"
```

Comment:
Do not modify existing code that works for normal chain !

Please revert everything here, have this query func included into sov run type chain

Reply:
+1